libraries/Microsoft.Bot.Builder.Prompts/ConfirmPrompt.cs

```diff
@@ -36,17 +39,100 @@ public string Text
     /// <summary>
     /// ConfirmPrompt recognizes confrimation expressions as bool 
     /// </summary>
-    public class ConfirmPrompt : BasePrompt<ConfirmResult>
+    public class ConfirmPrompt
```

Comment:
This is an interesting change. We'll need to involve @johnataylor  and @Stevenic  on this one.